src/api.js

```diff
@@ -756,11 +762,39 @@ API.prototype.router = function(options) {
         component:  options.component
       }));
     }
+    if (monitor) {
+      middleware.push( (req, res, next) => {
+        let sent = false;
+        let start = process.hrtime();
+        let send = () => {
+          try {
+            // Avoid sending twice
+            if (sent) {
+              return;
+            }
+            sent = true;
+
+            let d = process.hrtime(start);
+
+          for (let statusCode of [res.statusCode, 'all']) {
+            let k = entry.name + '.' + statusCode;
```

Comment:
I suggest we don't count for each status code... do we really want that...

Maybe we want to count number of successful, failed and 5xx requests, but do we need more?